Source/Api/Impl/AnalyticsClient.swift

```diff
@@ -18,31 +18,20 @@ import Foundation
 - Author: Allan Melo
 */
 internal class AnalyticsClient {
-    
-    init() {
-		let bundle = Bundle(for: type(of: self))
-		var settings: [String: String]?
+        
+	func send(endpointURL: String, analyticsEvents: AnalyticsEvents)
+		throws -> String {
 		
-		if let path = bundle.path(forResource: "settings", ofType:"plist") {
-			settings = NSDictionary(contentsOfFile: path) as? [String: String]
+		guard let url = URL(string: endpointURL) else {
+			throw HttpError.invalidUrl
 		}
-		
-		baseUrl = URL(string: settings?["ANALYTICS_GATEWAY"] ?? "")
-    }
-    
-	func send(analyticsEvents: AnalyticsEvents)
-		throws -> String {
-			
+            
 		let encoder = JSONEncoder()
 		encoder.dateEncodingStrategy = .iso8601
 		let analyticsData = try encoder.encode(analyticsEvents)
-			
-		guard let url = baseUrl else {
-			throw HttpError.invalidUrl
-		}
 		
 		let (data, _, error) = URLSession.sendPost(url: url, body: analyticsData)
-
+            
```

Comment:
There are some lines just with a tab added when it's not needed

Reply:
I'll fix that. Probably happens when my Xcode was not configured. Thanks